files/justfiles/utilities.just

```diff
@@ -159,7 +159,8 @@ install-steam:
         echo "Steam requires Xwayland, which your variant of secureblue has disabled by default."
```

Comment:
this is from a previous PR, but it just occurred to me that if for whatever reason someone is running a pure x11 session, this won't function properly. We should pgrep for `Xwayland || Xorg` i think